### ISSUE TITLE
[Tutorial] Reference to Insights in code example, but not present ASP.NET Core Mvc template

### ISSUE THREAD

**Issue by Powerhelmsman** [aspnet/Docs#9256]
This issue was created by @Xarkam from https://github.com/MicrosoftDocs/feedback/issues/748
Link to article:
https://docs.microsoft.com/fr-fr/aspnet/core/tutorials/first-mvc-app/adding-view?view=aspnetcore-2.1#change-the-title-and-menu-link-in-the-layout-file
Problem:
In code example of _layout.cshtml, you have @inject Microsoft.ApplicationInsights.AspNetCore.JavaScriptSnippet JavaScriptSnippet, but VisualStudio don't have any line or reference to Insights after initializing new mvc project
If you folow this tutorial and if you copy/paste the razor code example, you have error because, Insights is not present in asp.net core template.

**Comment by scottaddie** [comment]
@Xarkam The following directive in the *_Layout.cshtml* file is a remnant from the ASP.NET Core 1.x MVC project template:

```
@inject Microsoft.ApplicationInsights.AspNetCore.JavaScriptSnippet JavaScriptSnippet
```

You can safely remove it. It was removed from the 2.0 template and was used for Application Insights.

**Comment by Xarkam** [comment]
@scottaddie I know that I can remove it safely.

I have a friend who is learning C # and he did not understand why when he copied the sample code into the documentation, he could not build his project anymore.

It is by seeing this that I understood that the documentation is not up to date with the templates :)

**Comment by Rick-Anderson** [comment]
The sample was fixed in #9260
